### ISSUE TITLE
Make octgn use culture invariant.

### ISSUE THREAD

**Issue by Gravecorp** [octgn/OCTGN#1765]
If the region of the computer octgn is installed in uses a non standard datetime format in its regional settings it causes all sorts of weird errors and side effects.
Settings the current appdomain or the application to use culture.invariant should fix this and should only be a few lines of code to make it work application wide.
Alternative is to make it default to culture.US or something like that.

**Comment by Gravecorp** [comment]
that would be the most optimal solution but would require a ton of code for transforming culture sensitive objects to different formats especially when the formats come from outside like for instance the news or plugins. For the short term it would be much easier and faster to use the invariant or make .net auto convert it all to a single culture making it a much easier and reliable change. Single point of change versus a large amount of them with the chance we miss some.
But i will leave the decision up to you.

**Comment by kellyelton** [comment]
Making that change would add a dependency to that change for everything that relies on it. 

Also, it introduces a magic element and changes way more than just date formatting. 

What will happen is someone will end up struggle to debug why TextBlock is formatted incorrectly 10 months down the road in an obscure user built plugin.

I'd say in this case it makes more sense to go in and fix each specific problem instead of introduces new behavior.

**Comment by Gravecorp** [comment]
I agree with that we should implement language fixes or atleast a converter to compensate for weirdness. Its the better solution. In case this is not feasible because of time constraints the shortcut is setting it to invariant application wide. Just listing the options for this problem :P

**Comment by kellyelton** [comment]
Yeah that's a fair point. Maybe if do that, we should not close this
ticket, and plan for a day where we shut that off (if/when people get time
to help out)?

**Comment by Gravecorp** [comment]
Sounds like a plan ;)
